larray/inout/array.py

```diff
@@ -505,7 +505,7 @@ def read_excel(filepath, sheet=0, nb_axes=None, index_col=None, fill_value=np.na
     Parameters
     ----------
     filepath : str
-        Path where the Excel file has to be read.
+        Path where the Excel file has to be read or use -1 to refer to the last active workbook.
```

Comment:
s/last/currently/ ? I mean I don't want people to think it reopens closed workbooks.